src/Parser/Monk/Windwalker/Modules/Items/TheEmperorsCapacitor.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import SpellLink from 'common/SpellLink';
+import AbilityTracker from 'Parser/Core/Modules/AbilityTracker';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import Module from 'Parser/Core/Module';
+
+class TheEmperorsCapacitor extends Module {
+  static dependencies = {
+    abilityTracker: AbilityTracker,
+    combatants: Combatants,
+  };
+  totalStacks = 0;
+  currentStacks = 0;
+  stacksUsed = 0;
+  stacksWasted = 0;
+  cjlCasts = 0;
+  averageStacksUsed = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasChest(ITEMS.THE_EMPERORS_CAPACITOR.id);
+  }
+
+  on_byPlayer_applybuffstack(event) {
+    const spellId = event.ability.guid;
+    // You can only ever apply 1 stack at a time
+    if (SPELLS.THE_EMPERORS_CAPACITOR_STACK.id === spellId) {
+      this.totalStacks += 1;
+      if (this.currentStacks === 20) {
```

Comment:
Are you certain that an applybuffstack event is thrown when at max stacks?

Reply:
It does not, and refreshbuff doesn't either. I have however changed it so that it now checks for chi spenders in casts and adds a wasted stack there if the cap has been reached